src/test/tgen/fixed_duration/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# The same tgen server config is used for all tests.
+add_custom_target(tgen-duration-server-setup ALL
+                COMMAND ${CMAKE_COMMAND}
+                  -E copy
+                  ${CMAKE_CURRENT_SOURCE_DIR}/server.graphml
+                  ${CMAKE_CURRENT_BINARY_DIR}/server.graphml)
+
+foreach(client 1stream 10streams 100streams 1000streams)
+    add_custom_target(tgen-duration-${client}-setup ALL
+                    COMMAND ${CMAKE_COMMAND}
+                      -E copy
+                      ${CMAKE_CURRENT_SOURCE_DIR}/client.${client}.graphml
+                      ${CMAKE_CURRENT_BINARY_DIR}/client.${client}.graphml)
+
+  foreach(network 1mbit_300ms 10mbit_200ms 100mbit_100ms 1gbit_10ms)
+    add_custom_target(tgen-duration-${network}-${client}-setup ALL
```

Comment:
nit: maybe unindent the first `add_custom_target` by two spaces? it took me a little bit to understand what was going on with the loops here.